jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java

```diff
@@ -95,6 +95,13 @@ private List<TarArchiveEntry> getSortedEntries() {
     }
   }
 
+  private static void clearTimeHeaders(TarArchiveEntry entry) {
+    entry.addPaxHeader("mtime", "0");
```

Comment:
We do `entry.setModTime(layerEntry.getModificationTime().toEpochMilli())`. Should we set the same value? Or, is it just not worth it as a PAX header?

Reply:
I think this seems simpler tbh.